From 147d133aea71dfd605044132dd12bfb29c85cbd2 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Tue, 13 May 2025 15:42:32 +0100
Subject: [PATCH 1/1] compose: Insert screenshot tags for auto-generated font
 screenshots

Previously, compose would generate screenshot thumbnails for fonts in
the media directory but not actually insert screenshot tags into the
resulting font component.

Resolves #710.
---
 compose/asc-utils-fonts.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/compose/asc-utils-fonts.c b/compose/asc-utils-fonts.c
index 307ce60a..1830b892 100644
--- a/compose/asc-utils-fonts.c
+++ b/compose/asc-utils-fonts.c
@@ -158,6 +158,9 @@ asc_render_font_screenshots (AscResult *cres,
 
 			as_screenshot_add_image (scr, img);
 		}
+
+		if (as_screenshot_is_valid (scr))
+			as_component_add_screenshot (cpt, scr);
 	}
 
 	return TRUE;
-- 
2.49.0

